src/components/PollOptions/index.js

```diff
@@ -172,14 +172,14 @@ const styles = StyleSheet.create({
       top: 0,
       left: 0,
       borderRadius: 6,
-      backgroundColor: isMax ? COLORS.blueSea : COLORS.lightgrey
+      backgroundColor: isMax ? COLORS.blueSea : COLORS.gray100
     };
   },
   totalpolltext: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: 16,
-    color: COLORS.blackgrey
+    color: COLORS.gray400
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve updating the `backgroundColor` property from `COLORS.lightgrey` to `COLORS.gray100` and the `color` property from `COLORS.blackgrey` to `COLORS.gray400`. These are straightforward color changes and should not impact the functionality of the component.
```diff
-    backgroundColor: isMax ? COLORS.blueSea : COLORS.lightgrey
+    backgroundColor: isMax ? COLORS.blueSea : COLORS.gray100

-    color: COLORS.blackgrey
+    color: COLORS.gray400
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->